_posts/2019-03-12-so-you-want-to-be-an-engineer.md

```diff
@@ -0,0 +1,170 @@
+of it as a serious pursuit.
+
+Mostly, that was due to my experience programming in the past—I did about half a CS major in undergrad. At the
+time, I felt that programming wasn't right for me, and I dropped the major during my third year.
+
+It was Artsy's Engineering team that convinced me that programming was something that I both wanted to and could
+do. Our engineers have always welcomed learners and been happy to answer questions and empower other teams to do
+technical work. I eventually realized that the parts of my work where I was coding were the parts I enjoyed the
+most, and that I would likely feel more fulfilled if I made programming my full-time occupation.
+
+Here's what that journey looked like. Hopefully my experience proves helpful to you as you begin (or finish) yours!
+
+# Step One: Tell People What You Want
+
+This might've been the single biggest learning I took away from this experience: _if you tell people you want
+something, you might just get it._
+
+That may sound super obvious. It wasn't for me. I've usually been very passive in my career decisions, taking the
+path of least resistance and considering myself lucky when I was able to keep progressing. In this case, I was
+making a substantial departure from that idea by being proactive about what it was I wanted.
+[This post](https://engineering.gusto.com/i-didnt-want-to-be-a-manager-anymore-and-the-world-didnt-end/) by
+[Noa Elad](https://twitter.com/NoaElad) does a great job with this topic and is certainly worth a read.
+
+The first person I told at Artsy was [Orta](https://github.com/orta). He'd often encouraged me to develop my
+technical skills, and since he knows Artsy's engineering team and stack better than just about anyone, I figured
+he'd be able to point me in the right direction when it came to learning resources and navigating company politics
+to get to my eventual goal.
+
+The second person I told was my manager on the CRM team. I fortunately had a very good relationship with my manager
+and was confident that she would help me if she could. And by telling her early, I was giving her more opportunity
+to advocate for me and making it easier for her to replace me in the event that I was able to switch teams.
+
+The third person I told was Artsy's CTO, [dB](https://www.dblock.org/). This was Orta's recommendation—dB would be
+able to tell me if and when a move might be possible, and he could suggest things I should do to improve my chances
+of making the switch.
+
+I also didn't keep it a secret from the rest of my team or the company. I didn't show up wearing a shirt that said
+"ENGINEER" on it, but I told people, "I'm working on becoming an engineer. I'm really hoping to stay at Artsy, but
+if there's not a role open for me, that's fine—I'll search elsewhere."
+
+The net outcome of these conversations was that there wasn't a role open right then, but I also left with a better
```

Comment:
> there wasn't a role open right then 

The timeline here is a little unclear. 

> there wasn't a role open right then, but I still had a lot to learn anyway

might help clarify.

Reply:
good point—didn't use this exact language, but clarified in the most recent commit